pkg/arvo/sys/lull.hoon

```diff
@@ -1147,6 +1147,11 @@
   ::                                                    ::::
 ++  eyre  ^?
   |%
+  +$  cache-entry
+    $%
+      [%scry =path case=@ud]
```

Comment:
I think you should be able to get rid of both of these `case` variables, as they're now implicit in the list ordering in the axal data structure. After, that LGTM and we can get review from other core devs.

Reply:
hmm, like this?
```diff
         %scry
-      =/  res=(unit (unit cage))  (rof ~ %cx [our %base ud+case.cache-entry] path.cache-entry)
+      =/  case=@ud  (sub (lent cache-entries) 1)
+      =/  res=(unit (unit cage))  (rof ~ %cx [our %base ud+case] path.cache-entry)
       ?~  res    (error-response 500 "failed scry")
```
that seems wrong to me. I don't see why the clay case would necessarily match the cache case.

Reply:
Ah, I get what you're saying. The case has a different semantic than I thought it did. Let's just remove the one on the simple-payload, then.

Reply:
done